whoarder/clippings.py

```diff
@@ -96,13 +96,14 @@ def __next__(self):
         count = 1
         while True:
             if count > 5:
-                raise InvalidFormatException('''Input file doesn't seem to be
-                a clippings file, separators are missing or damaged''')
+                print(clipping_buffer[-1])
```

Comment:
@mynameismon probable debugging leftover; can you remove it?

Reply:
Apologies, removing right away